docs/ref/contrib/admin/index.txt

```diff
@@ -2468,6 +2527,12 @@ Templates can override or extend base admin templates as described in
 
     Path to a custom template that will be used by the admin site app index view.
 
+.. attribute:: AdminSite.empty_value_display
+
+    .. versionadded:: 1.9
+
+    Sets the string to use for displaying empty values in the admin site.
```

Comment:
chop "Sets"
"in the admin site change list. The value can be overridden on a per `ModelAdmin` or attribute basis using..."